framework/include/fvkernels/FVFluxKernel.h

```diff
@@ -32,6 +46,48 @@ class FVFluxKernel : public FVKernel,
   /// @}
 
 protected:
+  /// Provides interpolation of face values for non-advection-specific purposes
+  /// (although it can/will still be used by advective kernels sometimes).
+  /// This should be called when a face value needs to be computed using left
+  /// and right information (e.g. a material property, solution value, etc.).
+  /// left and right represent the property/value to compute the face value
+  /// for.
+  template <typename T>
+  T interpolate(InterpMethod m, T left, T right)
```

Comment:
I think we probably want `const T &` for the argument list because copy construction of `DualNumbers` will be expensive